pkg/component/bump_test.go

```diff
@@ -3,15 +3,14 @@ package component
 import (
 	"context"
 	"errors"
+	fakes "github.com/pivotal-cf/kiln/pkg/component/fakes_internal"
```

Comment:
Does package name "fakes_internal" still make sense in light of the fact that we're moving these to "pkg"?